index.js

```diff
@@ -1,5 +1,9 @@
 "use strict";
 
-const specs = require("./specs.json");
+const isString = obj =>
+  Object.prototype.toString.call(obj) === "[object String]";
+
+const specs = require("./specs.json")
+  .map(spec => isString(spec) ? { url: spec } : spec);
```

Comment:
Why not just `typeof spec === "string"`? The input here is JSON so there can't be any String objects.

Reply:
Good point, fixed.